sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/db/migration/V0169__StatusXmlToRunTable.java

```diff
@@ -0,0 +1,138 @@
+package uk.ac.bbsrc.tgac.miso.db.migration;
+
+import java.io.StringReader;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+import org.xml.sax.InputSource;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+/**
+ * Migration to remove a xml file from the MySql database. The useful values from the xml file are copied to fields in the database.
+ * 
+ * Flyway migration to pull four values out of an xml file stored in the xml field of the Status table. Four new columns are added to the
+ * Run table and the four values from the xml file are copied to the four new columns. The xml filed in the Status table is then deleted.
+ *
+ */
+public class V0169__StatusXmlToRunTable implements JdbcMigration {
+
+  @Override
+  public void migrate(Connection connection) throws Exception {
+    addStatusColumnsToRun(connection);
+    Map<String, String> statusXml = getStatusXml(connection);
+    Map<String, StatusValues> statusValues = statusXml.entrySet().stream()
+        .collect(Collectors.toMap(Map.Entry::getKey, entry -> new StatusValues(entry.getValue())));
+    updateStatusColumns(connection, statusValues);
+    deleteXmlColumnFromStatus(connection);
+  }
+
+  private Map<String, String> getStatusXml(Connection connection) throws SQLException {
+    Map<String, String> result = Maps.newHashMap();
+    PreparedStatement getStatusXml = connection.prepareStatement("SELECT s.runName, s.xml FROM Status AS s WHERE s.xml IS NOT NULL");
+    ResultSet rs = getStatusXml.executeQuery();
+    while (rs.next()) {
+      String key = rs.getString("runName");
+      String value = rs.getString("xml");
+      result.put(key, value);
+    }
+    return result;
+  }
+
+  private void addStatusColumnsToRun(Connection connection) throws SQLException {
+    List<String> newColumnSql = Lists.newArrayList();
+    newColumnSql.add("ALTER TABLE Run ADD COLUMN numCycles int(11) DEFAULT NULL;");
```

Comment:
These should go in a separate table `RunIllumina`. We could leave it like this and follow it up with a regular migration to massage the tables. In which case, let's call this 168 and that one 169.